### PR TITLE
Create service Account command support

### PR DESCRIPTION
**Pull request by tahmidefaz**
Creation Success:
image

Creation Failed:
image

### CODE REVIEW

File: src/types/Misc.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export enum EnvType {
```
Comment (justinorringer) [inline src/types/Misc.ts:1]: Misc isn't a great name either. Types wasn't good either though haha

**Review by justinorringer — APPROVED** [review]
Looking good! A little CSS would help like ya said. Maybe the bottom margin is too big?

File: src/types/Misc.ts (hunk 1 above)
Comment (tahmidefaz) [reply src/types/Misc.ts]: How about `Common`? Sounds slightly better than `Misc`, I think. :thinking:

Comment (justinorringer) [reply src/types/Misc.ts]: Sure that's better, I guess we'd get in the weeds of a bunch of variable files if they all are named specific

**Pull request merged**